tests/h/search/client_test.py

```diff
@@ -4,15 +4,28 @@
 
 import pytest
 
-from elasticsearch1 import RequestsHttpConnection
+import elasticsearch
+import elasticsearch1
+import elasticsearch_dsl
+import elasticsearch1_dsl
 
-from h.search.client import get_client
+from h.search.client import get_client, get_client_old
 from h.search.client import Client
 
 
-class TestClient(object):
+ES1_PATH = 'h.search.client.elasticsearch1.Elasticsearch'
```

Comment:
This stuff at the top could ultimately disappear once we are no longer testing two versions.